### PR TITLE
Fix SSR uglifyJs bug

### PR DESCRIPTION
**Pull request by Ivan-Feofanov**
Fix for issue #217

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,12 +34,10 @@ Vue.use(VueGoogleMaps, {
 
 For Nuxt.js projects, please import VueGoogleMaps in the following manner:
 ```js
-import * as VueGoogleMaps from '~/node_modules/vue2-google-maps/src/main'
+import * as VueGoogleMaps from '~/node_modules/vue2-google-maps'
```
Comment (homerjam) [inline README.md:37]: Does this still not work?
```
import * as VueGoogleMaps from 'vue2-google-maps'
```

**Comment by homerjam** [comment]
Good work on this.

Maybe you should remove `dist` changes from PR @Ivan-Feofanov ?

File: README.md (hunk 1 above)
Comment (Ivan-Feofanov) [reply README.md:37]: Only this way will work

**Comment by Ivan-Feofanov** [comment]
@homerjam the meaning of this PR is to rebuild `dist` folder

**Comment by diegoazh** [comment]
@Ivan-Feofanov the this folder will be change when a new version is released.

**Review by diegoazh — CHANGES_REQUESTED** [review]
@Ivan-Feofanov please remove the changes on `dist/` folder and resolve the conflicts in order to be merged.

**Comment by diegoazh** [comment]
I think that this PR should be closed for inactivity on Oct 27.

**Comment by diegoazh** [comment]
PR closed because the source branch of this change does not longer exist. Added to the readme file the proposal information.

**Pull request closed without merging**